src/ar100/spicmds.c

```diff
@@ -0,0 +1,141 @@
+// Commands for sending messages on an SPI bus
+//
```

Comment:
Why is this file copied from src/spicmds.c?  What's different about it and why can't the main spicmds.c be used?